libsolidity/analysis/ContractLevelChecker.cpp

```diff
@@ -151,10 +162,24 @@ void ContractLevelChecker::checkIllegalOverrides(ContractDefinition const& _cont
 			if (modifiers.count(name))
 				m_errorReporter.typeError(modifiers[name]->location(), "Override changes function to modifier.");
 
-			for (FunctionDefinition const* overriding: functions[name])
-				checkFunctionOverride(*overriding, *function);
-
-			functions[name].push_back(function);
+			if (function->isEtherReceiver())
```

Comment:
An alternative leading to simpler code would be to internally name them. The keywords are reserved anyway right?

Reply:
Suggested that to @chriseth in the chat at some point who suggested we should keep them unnamed - I don't mind either way.